content-src/components/Card/Card.jsx

```diff
@@ -245,10 +245,8 @@ export class _Card extends React.PureComponent {
           </div>
         </div>
       </a>
-      {!props.placeholder && <button aria-haspopup="true" className="context-menu-button icon" title={this.props.intl.formatMessage({id: "context_menu_title"})}
-        onClick={this.onMenuButtonClick}>
-        <span aria-haspopup="true" className="sr-only">{`Open context menu for ${link.title}`}</span>
-      </button>}
+      {!props.placeholder && <button aria-haspopup="true" aria-label={`Open context menu for ${link.title}`} className="context-menu-button icon" title={this.props.intl.formatMessage({id: "context_menu_title"})}
```

Comment:
We should fix this up to use `context_menu_button_sr=Open context menu for {title}` (but migrated to fluent 😉 ) instead of hardcoded.